src/storage/events/jwks-create-signing-secret.ts

```diff
@@ -0,0 +1,60 @@
+import { BaseEvent } from './base-event'
+import { Job, SendOptions, WorkOptions } from 'pg-boss'
+import { logger, logSchema } from '@internal/monitoring'
+import { BasePayload } from '@internal/queue'
+import { generateUrlSigningJwk } from '@internal/database'
+
+interface JwksCreateSigningSecretPayload extends BasePayload {
+  tenantId: string
+}
+
+export class JwksCreateSigningSecret extends BaseEvent<JwksCreateSigningSecretPayload> {
+  static queueName = 'tenants-jwks-create'
+
+  static getWorkerOptions(): WorkOptions {
```

Comment:
@fenos are these settings ok for this worker/queue? I copied these values (both `getWorkerOptions` and `getQueueOptions`) from `RunMigrationsOnTenants` let me know if you think they should be adjusted at all